.github/workflows/release.yaml

```
@@ -0,0 +1,127 @@
+
+    # env:  
+    #   BMF_REPO: ${{ vars.BMF_REPOSITORY_URL }}  
+    #   BMF_BRANCH: ${{ vars.BMF_BRANCH }}  
+    #   BMF_ROOT_PATH: ${{ github.workspace }}/bmf/output/bmf 
+
+    - name: Set up Qt
+      run: |
+        sudo apt-get install -y qt5-qmake qtbase5-dev qtchooser qtbase5-dev-tools cmake build-essential
+
+    - name: Build with CMake
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:49: Quote this to prevent word splitting

(shellcheck)